packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.html

```diff
@@ -49,6 +49,11 @@
         >
             <span data-translate="dedicatedCloud_tab_drp"></span>
         </oui-header-tabs-item>
+        <oui-header-tabs-item
+            data-ng-if="$ctrl.isNsxCompatible"
```

Comment:
Value doesn't change so you can use bind once
```suggestion
            data-ng-if=":: $ctrl.isNsxCompatible"
```